visidata/clipboard.py

```diff
@@ -58,6 +59,13 @@ def syscopyValue(sheet, val):
 
     vd.status('copied value to system clipboard')
 
+@Sheet.api
+def setColClipboard(sheet):
+    if not vd.memory.clipcells:
+        vd.warning("nothing to paste from clipcells")
+        return
```

Comment:
Whenever I see a `warning` followed by `return`, I wonder if it could be a `fail()` instead.  The difference being that any command which uses a function with `warning`/`return` will succeed, whereas with `fail` it will raise an exception and won't be added to the cmdlog, and will abort during a replay.  By default I think we should prefer failing, but if this case is either inconsequential, or potentially even useful in rare cases, then we would keep it as a `warning`/`return`.